tests/unit/adjlist.c

```diff
@@ -272,7 +272,7 @@ int test_multiedge_elimination_for_directed_graph(void) {
     return 0;
 }
 
-int test_caching() {
+int test_caching(void) {
```

Comment:
Hm, why did the lack of that `void` not cause a CI failure with the Clang 15 test?